pixel-painting/pixel-painting.cabal

```diff
@@ -0,0 +1,105 @@
+cabal-version: 2.2
+name:          pixel-painting
```

Comment:
small note, but we should probably rename the app the whole folder as `Hydraw`

Reply:
On it